specs/disciplina/witness/api/witness.yaml

```diff
@@ -264,6 +342,50 @@ paths:
           schema:
             $ref: "#/definitions/Hash"
 
+  /publications/educator/{educator}:
```

Comment:
Why not just add `includePubs` parameter, similarly to `includeTxs` one, to `/account/{address}` endpoint? Every Educator is an account and every account can be an Educator (in principle).
If you'd like to fetch transaction histories via separate endpoints, this makes sense (because, well, current approach with `includeTxs` is actually bullshit, because it doesn't provide any limiting/pagination). But then the most clean solution would be adding an `account` filter parameter to `/transaction` endpoint (or, if you'd like, `/transactions` and `/publications` endpoints).

Reply:
Let's assume I prefer a cleaner solution.
Just adding `account` filter parameter to `/transactions` is a bit difficult, because `/transactions` returns transactions with minimal info (`author`/`block`/`prevBlock` in `TxPublication` case), while `/publications` returns more elaborate private blocks (`fee`/`merkleRoot`/`transactions number`/`ATG`). Maybe this is a very bad and confusing difference.

Reply:
@Martoon-00 well, fair enough, that just means that because we don't have a proper way to encode sum types in JSON, the `/transactions` and `/publications` endpoints should be separate, and each of them should have the `account` filter parameter.

Reply:
@Martoon-00 just do for `/transactions` the same what `includeTxs` flag does in `/account` endpoint, but with limits.